person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = 1
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id, :parent_permission
+
+  private
+
+  def of_age
+    @age >= 18
+  end
+
+  def can_use_services
+    is_of_age && @parent_permission
+  end
+end
```

Comment:
- [ ] good job creating the `Person` class 👍 how the code is written now the `can_use_services` method is private, however, it is supposed to be public, this will be solved by placing it above the `private` keyword in line 12.